unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioClip.cs

```diff
@@ -95,8 +87,11 @@ IEnumerator TryToLoad()
                 Model model = (Model) this.model;
                 if (scene.contentProvider.HasContentsUrl(model.url))
                 {
-                    yield return Utils.FetchAudioClip(scene.contentProvider.GetContentsUrl(model.url),
-                        Utils.GetAudioTypeFromUrlName(model.url), OnComplete, OnFail);
+                    Utils.FetchAudioClip(
```

Comment:
Please keep the yield logic to minimize regression chances by the logic change.

Reply:
Reviewing the DCLAudioClip flow, I think the yield logic was uneeded in this case. Also, after several tests with audio, I didn't notice any problem. Anyway I will add it back (just in case). Done in https://github.com/decentraland/explorer/pull/2256/commits/d8a7c637eed3918d66ef0874dbd323b3d5cda549

Reply:
Ty!